src/braket/devices/local_simulator.py

```diff
@@ -134,12 +136,11 @@ def _run_internal(
 
 @_run_internal.register
 def _(circuit: Circuit, simulator: BraketSimulator, shots, *args, **kwargs):
-    if DeviceActionType.JAQCD not in simulator.properties.action:
+    if DeviceActionType.OPENQASM not in simulator.properties.action:
```

Comment:
I think we need to be careful about not introducing a breaking change here:
* pinning the local simulator to version >= (the one adding support for OpenQASM)
* the scenario where a customer is using another local simulator that they have developed. They might not want to support OpenQASM and yet still be able to use the LocalSimulator class to interact with their simulator.

Reply:
That's a good point. I think this is the case no matter how we approach this. If we flip the bdk to use OpenQASM by default, then it won't be compatible with a LocalSimulator that doesn't support OpenQASM. We could always launch the OpenQASM simulator and leave Jaqcd as the default, but I believe we want to switch over.

I think pinning the version is a good solution here. A customer developing their own local simulator will likely be able to figure out what's going on and resolve quickly

Reply:
Perhaps then we should merge the simulator repo a day earlier and release a new version so we can pin the dependency.

Reply:
Maybe another possibility is: 
```
if device supports OpenQASM:
  use OpenQASM
elif device supports JAQCD:
  use JAQCD
else: 
  raise error
```

Reply:
Oh right, good idea. Maybe I'll blame the covid brain fog for missing that option 😛 jk